pyne/mesh.py

```diff
@@ -1158,10 +1288,13 @@ def _structured_check(self):
             raise MeshError("Structured mesh methods cannot be called from "\
                             "unstructured mesh instances.")
 
-    def write_hdf5(self, filename):
+    def save(self, filename, write_mats=True):
+        self.write_hdf5(filename, write_mats)
+
+    def write_hdf5(self, filename, write_mats=True):
```

Comment:
Is there a reason why we have both save and write_hdf5 functions?

Reply:
It seems that this was a convenience given many places that called a `save` method on the previous mesh classes.

Reply:
Addressed in #62